### PR TITLE
Resolve duplicates for batch 2 mgmt plane SDK except Compute

### PR DESCRIPTION
**Pull request by Yao725**
Changed models in this PR include:
```text
Cdn:
SecretProperties (2):
    Azure.Security.KeyVault.Secrets (type Azure.Security.KeyVault.Secrets.SecretProperties)
    Azure.ResourceManager.Cdn (type Azure.ResourceManager.Cdn.Models.SecretProperties)

KeyVault:
NameAvailabilityReason (3):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.NameAvailabilityReason)
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.NameAvailabilityReason)
    Azure.ResourceManager.KeyVault (type Azure.ResourceManager.KeyVault.Models.NameAvailabilityReason)
NetworkRuleAction (2):
    Azure.ResourceManager.CognitiveServices (type Azure.ResourceManager.CognitiveServices.Models.NetworkRuleAction)
    Azure.ResourceManager.KeyVault (type Azure.ResourceManager.KeyVault.Models.NetworkRuleAction)

Compute:
PublicNetworkAccess (16): -> DiskPublicNetworkAccess 
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.PublicNetworkAccess)
    Azure.ResourceManager.AppConfiguration (type Azure.ResourceManager.AppConfiguration.Models.PublicNetworkAccess)
    Azure.ResourceManager.CognitiveServices (type Azure.ResourceManager.CognitiveServices.Models.PublicNetworkAccess)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.PublicNetworkAccess)
    Azure.ResourceManager.ContainerRegistry (type Azure.ResourceManager.ContainerRegistry.Models.PublicNetworkAccess)
    Azure.ResourceManager.ContainerService (type Azure.ResourceManager.ContainerService.Models.PublicNetworkAccess)
    Azure.ResourceManager.CosmosDB (type Azure.ResourceManager.CosmosDB.Models.PublicNetworkAccess)
    Azure.ResourceManager.DataFactory (type Azure.ResourceManager.DataFactory.Models.PublicNetworkAccess)
    Azure.ResourceManager.DeviceUpdate (type Azure.ResourceManager.DeviceUpdate.Models.PublicNetworkAccess)
    Azure.ResourceManager.IotCentral (type Azure.ResourceManager.IotCentral.Models.PublicNetworkAccess)
    Azure.ResourceManager.IotHub (type Azure.ResourceManager.IotHub.Models.PublicNetworkAccess)
    Azure.ResourceManager.MachineLearning (type Azure.ResourceManager.MachineLearning.Models.PublicNetworkAccess)
    Azure.ResourceManager.Media (type Azure.ResourceManager.Media.Models.PublicNetworkAccess)
    Azure.ResourceManager.Redis (type Azure.ResourceManager.Redis.Models.PublicNetworkAccess)
    Azure.ResourceManager.Relay (type Azure.ResourceManager.Relay.Models.PublicNetworkAccess)
    Azure.ResourceManager.Storage (type Azure.ResourceManager.Storage.Models.PublicNetworkAccess)
NetworkProfile (4): -> VirtualMachineNetworkProfile 
    Azure.ResourceManager.AppPlatform (type Azure.ResourceManager.AppPlatform.Models.NetworkProfile)
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.NetworkProfile)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.NetworkProfile)
    Azure.ResourceManager.Workloads (type Azure.ResourceManager.Workloads.Models.NetworkProfile)
ResourceSkuRestrictionsReasonCode (4):
    Azure.ResourceManager.AppPlatform (type Azure.ResourceManager.AppPlatform.Models.ResourceSkuRestrictionsReasonCode)
    Azure.ResourceManager.CognitiveServices (type Azure.ResourceManager.CognitiveServices.Models.ResourceSkuRestrictionsReasonCode)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ResourceSkuRestrictionsReasonCode)
    Azure.ResourceManager.StoragePool (type Azure.ResourceManager.StoragePool.Models.ResourceSkuRestrictionsReasonCode)
ResourceSkuRestrictionsType (4):
    Azure.ResourceManager.AppPlatform (type Azure.ResourceManager.AppPlatform.Models.ResourceSkuRestrictionsType)
    Azure.ResourceManager.CognitiveServices (type Azure.ResourceManager.CognitiveServices.Models.ResourceSkuRestrictionsType)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ResourceSkuRestrictionsType)
    Azure.ResourceManager.StoragePool (type Azure.ResourceManager.StoragePool.Models.ResourceSkuRestrictionsType)
OSProfile (4): VirtualMachineOSProfile
    Azure.ResourceManager.ArcScVmm (type Azure.ResourceManager.ArcScVmm.Models.OSProfile)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.OSProfile)
    Azure.ResourceManager.ConnectedVMwarevSphere (type Azure.ResourceManager.ConnectedVMwarevSphere.Models.OSProfile)
    Azure.ResourceManager.Workloads (type Azure.ResourceManager.Workloads.Models.OSProfile)
StorageProfile (4): VirtualMachineStorageProfile 
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.StorageProfile)
    Azure.ResourceManager.ConnectedVMwarevSphere (type Azure.ResourceManager.ConnectedVMwarevSphere.Models.StorageProfile)
    Azure.ResourceManager.MySql (type Azure.ResourceManager.MySql.Models.StorageProfile)
    Azure.ResourceManager.PostgreSql (type Azure.ResourceManager.PostgreSql.Models.StorageProfile)
ResourceSkuZoneDetails (3): Compute
    Azure.ResourceManager.AppPlatform (type Azure.ResourceManager.AppPlatform.Models.ResourceSkuZoneDetails)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ResourceSkuZoneDetails)
    Azure.ResourceManager.StoragePool (type Azure.ResourceManager.StoragePool.Models.ResourceSkuZoneDetails)
HardwareProfile (3): VirtualMachineHardwareProfile 
    Azure.ResourceManager.ArcScVmm (type Azure.ResourceManager.ArcScVmm.Models.HardwareProfile)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.HardwareProfile)
    Azure.ResourceManager.ConnectedVMwarevSphere (type Azure.ResourceManager.ConnectedVMwarevSphere.Models.HardwareProfile)
ImageReference (3): wont change
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.ImageReference)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ImageReference)
    Azure.ResourceManager.Workloads (type Azure.ResourceManager.Workloads.Models.ImageReference)
OperatingSystemType (3): wont change
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.OperatingSystemType)
    Azure.ResourceManager.ContainerInstance (type Azure.ResourceManager.ContainerInstance.Models.OperatingSystemType)
    Azure.ResourceManager.MachineLearning (type Azure.ResourceManager.MachineLearning.Models.OperatingSystemType)
ResourceSkuCapacityScaleType (2): Compute
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.ResourceSkuCapacityScaleType)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ResourceSkuCapacityScaleType)
ResourceSkuCapabilities (2): Compute
    Azure.ResourceManager.AppPlatform (type Azure.ResourceManager.AppPlatform.Models.ResourceSkuCapabilities)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ResourceSkuCapabilities)
CachingType (2): wont change
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.CachingType)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.CachingType)
DataDisk (2): VirtualMachineDataDisk
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.DataDisk)
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.DataDisk)
EncryptionType (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.EncryptionType)
    Azure.ResourceManager.ElasticSan (type Azure.ResourceManager.ElasticSan.Models.EncryptionType)
LinuxConfiguration (2): wont change
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.LinuxConfiguration)
    Azure.ResourceManager.Workloads (type Azure.ResourceManager.Workloads.Models.LinuxConfiguration)
LoadBalancerConfiguration (2): CloudServiceLoadBalancerConfiguration
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.LoadBalancerConfiguration)
    Azure.ResourceManager.SqlVirtualMachine (type Azure.ResourceManager.SqlVirtualMachine.Models.LoadBalancerConfiguration)
LogAnalytics (2): wont change first come first serve
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.LogAnalytics)
    Azure.ResourceManager.ContainerInstance (type Azure.ResourceManager.ContainerInstance.Models.LogAnalytics)
ProtocolType (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ProtocolType)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.ProtocolType)
PublicIPAddressSku (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.PublicIPAddressSku)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.PublicIPAddressSku)
PublicIPAddressSkuName (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.PublicIPAddressSkuName)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.PublicIPAddressSkuName)
PublicIPAddressSkuTier (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.PublicIPAddressSkuTier)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.PublicIPAddressSkuTier)
ReplicationMode (2): GalleryReplicationMode
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ReplicationMode)
    Azure.ResourceManager.Sql (type Azure.ResourceManager.Sql.Models.ReplicationMode)
ReplicationState (2): RegionalReplicationState 
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.ReplicationState)
    Azure.ResourceManager.Sql (type Azure.ResourceManager.Sql.Models.ReplicationState)
RunCommandResult (2): VirtualMachineRunCommandResult
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.RunCommandResult)
    Azure.ResourceManager.ContainerService (type Azure.ResourceManager.ContainerService.Models.RunCommandResult)
SecurityProfile (2): wont change first come first serve
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.SecurityProfile)
    Azure.ResourceManager.HDInsight (type Azure.ResourceManager.HDInsight.Models.SecurityProfile)
StatusLevelType (2): Compute
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.StatusLevelType)
    Azure.ResourceManager.ConnectedVMwarevSphere (type Azure.ResourceManager.ConnectedVMwarevSphere.Models.StatusLevelType)
UpgradeMode (2): VirtualMachineScaleSetUpgradeMode
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.UpgradeMode)
    Azure.ResourceManager.ServiceFabric (type Azure.ResourceManager.ServiceFabric.Models.UpgradeMode)
WindowsConfiguration (2): first come first serve
    Azure.ResourceManager.Compute (type Azure.ResourceManager.Compute.Models.WindowsConfiguration)
    Azure.ResourceManager.Workloads (type Azure.ResourceManager.Workloads.Models.WindowsConfiguration)

Network
ConnectionStatus (5):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.ConnectionStatus)
    Azure.ResourceManager.AppConfiguration (type Azure.ResourceManager.AppConfiguration.Models.ConnectionStatus)
    Azure.ResourceManager.ContainerRegistry (type Azure.ResourceManager.ContainerRegistry.Models.ConnectionStatus)
    Azure.ResourceManager.ContainerService (type Azure.ResourceManager.ContainerService.Models.ConnectionStatus)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.ConnectionStatus)
HttpHeader (4):
    Azure.Core (type Azure.Core.HttpHeader)
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.HttpHeader)
    Azure.ResourceManager.ContainerInstance (type Azure.ResourceManager.ContainerInstance.Models.HttpHeader)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.HttpHeader)
IssueType (3):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.IssueType)
    Azure.ResourceManager.AppService (type Azure.ResourceManager.AppService.Models.IssueType)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.IssueType)
PrivateLinkServiceConnection (3):
    Azure.ResourceManager.DeviceUpdate (type Azure.ResourceManager.DeviceUpdate.Models.PrivateLinkServiceConnection)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.PrivateLinkServiceConnection)
    Azure.ResourceManager.StreamAnalytics (type Azure.ResourceManager.StreamAnalytics.Models.PrivateLinkServiceConnection)
ConnectivityHop (2):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.ConnectivityHop)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.ConnectivityHop)
ConnectivityIssue (2):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.ConnectivityIssue)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.ConnectivityIssue)
PreferredIPVersion (2):
    Azure.ResourceManager.ApiManagement (type Azure.ResourceManager.ApiManagement.Models.PreferredIPVersion)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.PreferredIPVersion)
InboundNatPool (2):
    Azure.ResourceManager.Batch (type Azure.ResourceManager.Batch.Models.InboundNatPool)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.InboundNatPool)
TagsObject (2):
    Azure.ResourceManager.ContainerService (type Azure.ResourceManager.ContainerService.Models.TagsObject)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.TagsObject)
EndpointType (2):
    Azure.ResourceManager.HybridConnectivity (type Azure.ResourceManager.HybridConnectivity.Models.EndpointType)
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.EndpointType)
ConnectionState (2):
    Azure.ResourceManager.Network (type Azure.ResourceManager.Network.Models.ConnectionState)
    Azure.ResourceManager.Relay (type Azure.ResourceManager.Relay.Models.ConnectionState)
```
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 1]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
   DedicatedHostGroup.properties.hosts: DedicatedHosts
   UefiSettings.secureBootEnabled: IsSecureBootEnabled
   UefiSettings.vTpmEnabled: IsVirtualTpmEnabled
+  PublicNetworkAccess: DiskPublicNetworkAccess
+  NetworkProfile: VmNetworkProfile
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:156]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 2]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  ResourceSkuRestrictionsReasonCode: ComputeResourceSkuRestrictionsReasonCode
+  ResourceSkuRestrictionsType: ComputeResourceSkuRestrictionsType
+  OSProfile: VmOSProfile
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:159]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 3]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  StorageProfile: DiskStorageProfile
+  ResourceSkuZoneDetails: ComputeResourceSkuZoneDetails
+  HardwareProfile: VmHardwareProfile
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:162]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 4]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  EncryptionType: DataDiskEncryptionType
+  LinuxConfiguration: LinuxOSConfiguration
+  LoadBalancerConfiguration: CloudServiceLoadBalancerConfiguration
+  ProtocolType: WinRMListenerProtocolType
+  PublicIPAddressSku: ComputePublicIPAddressSku
+  PublicIPAddressSkuName: ComputePublicIPAddressSkuName
+  PublicIPAddressSkuTier: ComputePublicIPAddressSkuTier
+  ReplicationMode: GalleryArtifactReplicationMode
+  ReplicationState: RegionalReplicationState
+  RunCommandResult: VmRunCommandResult
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:178]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 5]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  RunCommandInput: VmRunCommandInput
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:179]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 6]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  SecurityProfile: VmSecurityProfile
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:180]: We decided not use abbreviation for `VirtualMachine` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 7]
```diff
@@ -152,6 +152,35 @@ rename-mapping:
...
+  StatusLevelTypes: StatusLevelCode
+  UpgradeMode: VmssUpgradeMode
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:182]: We decided not use abbreviation for `VirtualMachineScaleSet` but use full name

File: sdk/compute/Azure.ResourceManager.Compute/src/autorest.md [hunk 8]
```diff
@@ -125,7 +125,7 @@ rename-mapping:
   SubResourceWithColocationStatus: ComputeSubResourceDataWithColocationStatus
```
Comment (ArthurMa1978) [inline sdk/compute/Azure.ResourceManager.Compute/src/autorest.md:125]: Dapeng is also working on the `Comtpute`, please make sure merge is correct

Comment (Yao725) [reply sdk/compute/Azure.ResourceManager.Compute/src/autorest.md]: Offline talked with Dapeng, this PR will only cover all the batch 2 RPs except compute, so I just reverted the change in compute.

**Comment by Yao725** [comment]
This PR will not include the change of compute to avoid too many conflicts.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**